### ISSUE TITLE
Multiple problems with "pxrj" output

### ISSUE THREAD

**Issue by jduck** [radare/radare2#4909]
Apart from the relatively minor issue that I PR'd in #4908, there are several other problems with "pxrj".
1. "pxrj > output" and "pxrj | cat > output" differ! For example:

```
-{"addr":140723317684320,"value":140723317688018,"ref":"stack R W 0x524f4c4f435f534c (LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:su=37;41:sg=30;43:ca=30;41LS_COLORS=rs=0:di=01;34:ln=01;36) --> ascii"},
+{"addr":140723317684320,"value":140723317688018,"ref":"stack R W 0x524f4c4f435f534c (LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:su=37;41:sg=30;43:ca=30;41LS_COLORS=rs=0:di=01;34:ln=01;36\x12\x05\x1=\x/85\x7?) --> ascii"},
```
1. the output appears to have repetition or termination issues. from above, you can see "LS_COLORS" within the value of "LS_COLORS". this is absolutely incorrect.

On a side note, what is the string length limit? Is there such a setting? Does it just use whatever available space on the current line?

**Comment by radare** [comment]
this happens (probably) because pxrj is constructed by calling commands in commands in commands, which results in broken rcons buffer , this can be fixed by using the api or using rcons.push/pop everytime this subcommand is called

**Comment by alvarofe** [comment]
Fixed in master e97c2cf55e4ab44cb54036b1e23a9823f4613705
